components/automate-cli/cmd/chef-automate/verify.go

```diff
@@ -454,7 +454,6 @@ func (v *verifyCmdFlow) runVerifyServiceForRemote(batchCheckConfig models.Config
 	}
 
 	if len(hostIPsToCopyLatestCLI) > 0 {
-		v.Writer.Printf("Copying CLI to the follwoing IPs: %v\n", hostIPsToCopyLatestCLI)
```

Comment:
Why do we need to remove this ?

Reply:
We are already showing a generic message in case of copying the CLI to remote nodes
![image](https://github.com/chef/automate/assets/54614142/bba080c7-6f7e-4e27-8463-c246392817c6)

Reply:
ok